src/main/java/org/mule/extension/db/internal/operation/DmlOperations.java

```diff
@@ -244,14 +246,18 @@ public int delete(@ParameterGroup(name = QUERY_GROUP) QueryDefinition query,
 
     QueryStatementFactory statementFactory = getStatementFactory(call);
 
-    InsensitiveMapRowHandler recordHandler = new InsensitiveMapRowHandler(connection, connector.getCharset());
+    Charset charset = connector.getCharset();
+    InsensitiveMapRowHandler recordHandler = new InsensitiveMapRowHandler(connection, charset);
 
     StatementStreamingResultSetCloser resultSetCloser = new StatementStreamingResultSetCloser(connection);
     flowListener.onComplete(new ResultSetCloserRunnable(resultSetCloser));
 
-    StatementResultHandler resultHandler = connection.getJdbcConnection().getMetaData().supportsMultipleOpenResults()
```

Comment:
Any reason to change the ternary operator to the if statement?

Reply:
I hate ternary operators :p